### PR TITLE
Adding sts connector as a feature

### PR DESCRIPTION
**Pull request by mushthaq33**
### Proposed changes in this pull request

Adding sts connector as a feature

### CODE REVIEW

File: features/org.wso2.carbon.sts.connector.feature/pom.xml [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+  ~  specific language governing permissions and limitations
+  ~  under the License.
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <parent>
+        <groupId>org.wso2.carbon.identity.inbound.auth.sts</groupId>
+        <artifactId>identity-inbound-auth-sts</artifactId>
+        <relativePath>../../pom.xml</relativePath>
+        <version>5.6.4</version>
```
Comment (sachiniWettasinghe) [inline features/org.wso2.carbon.sts.connector.feature/pom.xml:25]: the version should be `5.6.5-SNAPSHOT`

File: features/org.wso2.carbon.sts.connector.feature/resources/p2.inf [hunk 2]
```diff
@@ -0,0 +1,5 @@
+instructions.configure = \
+org.eclipse.equinox.p2.touchpoint.natives.mkdir(path:${installFolder}/../../deployment/);\
+org.eclipse.equinox.p2.touchpoint.natives.mkdir(path:${installFolder}/../../deployment/client/);\
+org.eclipse.equinox.p2.touchpoint.natives.mkdir(path:${installFolder}/../../deployment/client/modules/);\
+org.eclipse.equinox.p2.touchpoint.natives.copy(source:${installFolder}/../features/org.wso2.carbon.sts.connector_${feature.version}/rampart-1.6.1-wso2v43.mar,target:${installFolder}/../../deployment/client/modules/rampart-1.6.1-wso2v43.mar,overwrite:true);\
```
Comment (sachiniWettasinghe) [inline features/org.wso2.carbon.sts.connector.feature/resources/p2.inf:5]: Why can't we define the rampart version here as `rampart.wso2.version`?

Comment (sachiniWettasinghe) [inline features/org.wso2.carbon.sts.connector.feature/resources/p2.inf:5]: Add new line EOF

Comment (mushthaq33) [reply features/org.wso2.carbon.sts.connector.feature/resources/p2.inf:5]: Tried to add the version as rampart.wso2.version but it doesn't pick it up.

Comment (mushthaq33) [reply features/org.wso2.carbon.sts.connector.feature/resources/p2.inf]: Fixed with 5ddc88a9243e1fe9152f30a49d41b73168ffc63b

File: features/org.wso2.carbon.sts.connector.feature/pom.xml (hunk 1 above)
Comment (mushthaq33) [reply features/org.wso2.carbon.sts.connector.feature/pom.xml]: Fixed with 5ddc88a9243e1fe9152f30a49d41b73168ffc63b

**Review by sachiniWettasinghe — APPROVED** [review]

**Pull request merged**